test/test_files/log_files/samus_returns/door_lock.rdvgame

```diff
@@ -508,7 +508,7 @@
                     "type": "door",
                     "name": "Power Beam Door"
                 },
-                "Area 1/Water Maze/Door to Gullugg Gangway": {
+                "Area 1/Water Maze/Door to Gullugg Gangway (Bottom)": {
```

Comment:
revert this change. the migration should handle the test not failing, not the modification of the rdvgame.